apps/builddao/widget/Compose.jsx

```diff
@@ -157,6 +155,7 @@ const postToCustomFeed = ({ feed, text, labels }) => {
       setPostContent("");
       Storage.privateSet(draftKey, props.template || "");
       setHandler("autocompleteSelected"); // this is a hack to force the iframe to update
+      setComposeKey(generateUID());
```

Comment:
How come this? How would you remember the uuid of your draft to revisit it?

Reply:
Oh, this is just there to trigger a rerender of the compose iframe. Usually the iframe does not rerender and causes the iframe to still contain the post content after posting. This fixes that

Reply:
The uuid and drafts are different. The UUID is for the iframe only just to trigger a rerender